### ISSUE TITLE
LR during training

### ISSUE THREAD

**Issue by harsanyika** [iro-cp/FCRN-DepthPrediction#38]
In the article you mention, that the initial the learning rate is 0.01 for all layers. I would like to know if you used equal learning rates for all layers during the training or just at the beginning. In other words, when you reduce the learning rates during training, do you reduce it with the same amount for every layer?

**Comment by chrirupp** [comment]
The learning rate is equal for all layers. It is reduced everywhere.

**Comment by harsanyika** [comment]
How much do you reduce the learning rate when you observe a plateau in the validation loss?

**Comment by chrirupp** [comment]
We usually went from 0.01 to 0.005 to 0.001 and so on. Basically alternating between 1/2 and 1/5.

**Comment by mortalsoulzy** [comment]
@chrirupp Hi，sorry to disturb you but I'm quite confused when reproducing the experiment with tensorflow ，could you please share the training code on github？That would be of great help,Tanks you~

**Comment by AI-slam** [comment]
@chrirupp @harsanyika,  I am confused about how to validate the loss when observe a plateau, If you pick some data from train dataset and test loss during training, which loss you use?

**Comment by harsanyika** [comment]
@AI-slam. Its pretty straightforward I think. After a training epoch was finished, I checked the BerHu loss on the full validation dataset. When I observed that the validation loss stopped improving, I decreased the LR.
